internal/component/kafka/sasl_oauthbearer.go

```diff
@@ -107,7 +107,8 @@ func (ts *OAuthTokenSource) Token() (*sarama.AccessToken, error) {
 
 	oidcCfg := ccred.Config{ClientID: ts.ClientID, ClientSecret: ts.ClientSecret, Scopes: ts.Scopes, TokenURL: ts.TokenEndpoint.TokenURL, AuthStyle: ts.TokenEndpoint.AuthStyle}
 
-	timeoutCtx, _ := ctx.WithTimeout(ctx.TODO(), tokenRequestTimeout) //nolint:govet
+	timeoutCtx, cancel := ctx.WithTimeout(ctx.TODO(), tokenRequestTimeout) //nolint:govet
```

Comment:
Remove the `nolint` directive now

Reply:
To be clear, our linter fails because this directive is here

> Error: directive `//nolint:govet` is unused for linter "govet" (nolintlint)